kdb/key.go

```diff
@@ -1,5 +1,6 @@
 package kdb
 
+// TODO REVIEW: cleanup?
```

Comment:
What do you mean by cleanup?

Reply:
That the code below would be a candidate to be removed.

Reply:
This is the CGO preamble with the #include directives, it cannot be removed

Reply:
Ahh, ok. Please write this as comment above/below. Do we really need stdlib.h?

Reply:
Yes we need to `free()` the pointers

Reply:
I would prefer not commenting the preamble because people that want to work on these bindings should atleast have *some* knowledge of CGO and this is literally the first thing that is mentioned in the documentation. Also we would have to comment this in every file that uses CGO (at least 3 files).

Reply:
I did not see any reference that you used CGO. Can you please state that in the docu.

Reply:
`import "C"` is the reference to CGO - I will add some documentation to the readme!